Packs/Core/Playbooks/Playbook-Large_Upload.yml

```diff
@@ -0,0 +1,3868 @@
+          "x": -220,
+          "y": 4740
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "48":
+    id: "48"
+    taskid: eb7ba95d-5baf-464f-82e2-a47bb61b4f46
+    type: condition
+    task:
+      id: eb7ba95d-5baf-464f-82e2-a47bb61b4f46
+      version: -1
+      name: Found Related Indicators to Hunt?
+      description: Determine whether the endpoint should be isolated automatically.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      "yes":
+      - "55"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: inputs.AutoIsolateEndpoint
+            iscontext: true
+          right:
+            value:
+              simple: "True"
+          ignorecase: true
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: PaloAltoNetworksXDR.OriginalAlert._all_events.agent_id
+            iscontext: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 210,
+          "y": 4740
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "55":
+    id: "55"
+    taskid: 515db6e3-486b-4a62-8bec-02491d2bc3a6
+    type: regular
+    task:
+      id: 515db6e3-486b-4a62-8bec-02491d2bc3a6
+      version: -1
+      name: Core Isolate Endpoint
+      description: Isolates the specified endpoint.
+      script: '|||core-isolate-endpoint'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "7"
+    scriptarguments:
+      endpoint_id:
+        simple: ${PaloAltoNetworksXDR.OriginalAlert._all_events.agent_id}
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 210,
+          "y": 4890
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+view: |-
+  {
+    "linkLabelsPosition": {
+      "14_20_#default#": 0.5,
+      "17_20_#default#": 0.16,
+      "19_20_#default#": 0.11,
+      "22_23_yes": 0.46,
+      "27_34_#default#": 0.28,
+      "31_34_#default#": 0.21,
+      "32_34_#default#": 0.25,
+      "39_46_Remediation": 0.18,
+      "39_7_#default#": 0.11,
+      "3_6_#default#": 0.34,
+      "4_7_#default#": 0.18,
+      "5_6_Yes": 0.13,
+      "5_7_#default#": 0.1,
+      "8_7_#default#": 0.1
+    },
+    "paper": {
+      "dimensions": {
+        "height": 5235,
+        "width": 2770,
+        "x": -550,
+        "y": -110
+      }
+    }
+  }
+inputs:
+- key: InternalIPRanges
+  value: {}
```

Comment:
are we not supporting a list here? Should we add the default value as mentioned in the description?

Reply:
added the default value as mentioned in the description